### PR TITLE
Fix #211 Add support for setConfirmationRequired for Android

### LINKED ISSUE #211: Add support for setConfirmationRequired for Android
Just a simple request to expose BiometricPrompt.PromptInfo builder option setConfirmationRequired for Android.

The reason I ask is that for Pixel 4 every time the prompt comes up there is a confirmation button:

image

Thanks

### PR DESCRIPTION
**Pull request by mibrito707**
# Description
Add support for `setConfirmationRequired` for Android
# How did you test your changes?
On a real device. Pixel 4 + Android 10

Closes #211

### CODE REVIEW

**Comment by mibrito707** [comment]
Hey @NiklasMerz 
Yes, you're seeing it correctly, the default is `true`. 
I just followed the default value specified in the official [doc](https://developer.android.com/reference/androidx/biometric/BiometricPrompt.PromptInfo.Builder.html#setConfirmationRequired(boolean)) "**BiometricPrompt will require confirmation by default.**"

**Comment by exxbrain** [comment]
What about iOS?

File: README.md [hunk 1]
```diff
@@ -117,6 +117,7 @@ Fingerprint.show({
      * Android: `"Use Backup"` (Because backup could be anything pin/pattern/password ..haven't figured out a reliable way to determine lock type yet [source](https://stackoverflow.com/questions/7768879/check-whether-lock-was-enabled-or-not/18720287))
 * __disableBackup__: If `true` remove backup option on authentication dialogue. Default: `false`. This is useful if you want to implement your own fallback.
 * __cancelButtonTitle__: For cancel button on Android
+* __confirmationRequired__: If `false` user confirmation is NOT required after a biometric has been authenticated on Android. Default: `true`. See [docs](https://developer.android.com/training/sign-in/biometric-auth#no-explicit-user-action).
```
Comment (NiklasMerz) [inline README.md:120]:
```suggestion
* __confirmationRequired__ (**Android**): If `false` user confirmation is NOT required after a biometric has been authenticated . Default: `true`. See [docs](https://developer.android.com/training/sign-in/biometric-auth#no-explicit-user-action).
```

**Comment by NiklasMerz** [comment]
> What about iOS?

I was wondering that too. Looks like there is no API for that yet.

**Pull request merged**